### PR TITLE
COM-8782 Fix password in fluendo-cerbero repo

### PR DESCRIPTION
**Pull request by mjimenofluendo**

### CODE REVIEW

File: cerbero/build/fridge.py [hunk 1]
```diff
@@ -85,6 +87,20 @@ def __init__(self, remote, username='', password=''):
...
+    def fromNetRc(cls, remote):
+        if "/" in remote:
+            remotePathParts = remote.split("/")
+            remoteHost = remotePathParts[0]
+        else:
+            remoteHost = remote
+        if os.name == 'posix':
+            netrc_file = expanduser("~/.netrc")
+        else:
+            netrc_file = os.path.expanduser(os.path.join('%USERPROFILE%', '.netrc'))
```
Comment (rgonzalezfluendo) [inline cerbero/build/fridge.py:100]: Not needed: https://github.com/python/cpython/blob/main/Lib/netrc.py#L67-L70

**Review by cfoch — CHANGES_REQUESTED** [review]

File: cerbero/build/fridge.py [hunk 2]
```diff
@@ -85,6 +85,16 @@ def __init__(self, remote, username='', password=''):
     def __str__(self):
         return 'remote \'{}\', username \'{}\', password \'{}\''.format(self.remote, self.username, self.password)
 
+    @classmethod
+    def fromNetRc(cls, remote):
```
Comment (cfoch) [inline cerbero/build/fridge.py:89]: nitpick: snake case, as the rest of the code here

**Review by cfoch — CHANGES_REQUESTED** [review]

File: cerbero/build/fridge.py [hunk 3]
```diff
@@ -85,6 +85,16 @@ def __init__(self, remote, username='', password=''):
...
+        if "/" in remote:
```
[the remark this answers is not in the document]
Comment (mjimenofluendo) [reply cerbero/build/fridge.py]: done

File: cerbero/build/fridge.py (hunk 2 above)
Comment (mjimenofluendo) [reply cerbero/build/fridge.py]: done

**Review by cfoch — APPROVED** [review]
LGTM

File: cerbero/build/fridge.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (rgonzalezfluendo) [reply cerbero/build/fridge.py]:
```
print(urlparse('officestorage1.fluendo.lan/data/fluendo/tech/public/cerbero-binaries'))
ParseResult(scheme='', netloc='', path='officestorage1.fluendo.lan/data/fluendo/tech/public/cerbero-binaries', params='', query='', fragment='')
```

**Pull request merged**